crc32c fails to build on amd64

The package crc32c with version 1.1.2 and epoch 2 fails to build from source on amd64 with the following error:

```
2025-07-07 07:30:25.382633185 +0000 UTC parsed env
2025-07-07 07:30:25.382997779 +0000 UTC configuring puller identity "720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69"...
2025-07-07 07:30:25.383718968 +0000 UTC running command chainctl [auth login --audience apk.cgr.dev --identity 720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69]
2025-07-07 07:30:25.674366363 +0000 UTC Successfully exchanged token.
2025-07-07 07:30:25.765310331 +0000 UTC Valid! Id: 720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69
2025-07-07 07:30:25.820746295 +0000 UTC Updates are available for chainctl (current version: 0.2.95; latest: 0.2.99). To install, please run:
2025-07-07 07:30:25.820749434 +0000 UTC     $ chainctl update
2025-07-07 07:30:25.825305405 +0000 UTC command "chainctl" completed successfully
2025-07-07 07:30:25.825384906 +0000 UTC puller identity configured successfully
2025-07-07 07:30:25.82539428 +0000 UTC puller identity configured successfully
2025-07-07 07:30:25.825401113 +0000 UTC running build...
2025-07-07 07:30:25.8254087 +0000 UTC running command melange [build crc32c.yaml --gcplog --source-dir crc32c --arch x86_64 --arch=x86_64 --env-file=build-x86_64.env --namespace=wolfi --pipeline-dir=pipelines --out-dir=./packages/ --runner=qemu --keyring-append=https://packages.wolfi.dev/os/wolfi-signing.rsa.pub --repository-append=https://packages.wolfi.dev/os --repository-append=https://apk.cgr.dev/wolfi-presubmit/world-1751869916414 --repository-append=https://apk.cgr.dev/wolfi-presubmit/world-1751869916414 --git-repo-url=https://github.com/wolfi-dev/os]
2025-07-07 07:30:25.866665456 +0000 UTC melange v0.28.0+dirty is building:
2025-07-07 07:30:25.878216618 +0000 UTC populating workspace /tmp/melange-workspace-400081569 from crc32c
2025-07-07 07:30:26.049833322 +0000 UTC Updates are available for chainctl (current version: 0.2.95; latest: 0.2.99). To install, please run:
2025-07-07 07:30:26.049837087 +0000 UTC     $ chainctl update
2025-07-07 07:30:27.720130501 +0000 UTC image configuration:
2025-07-07 07:30:27.720242395 +0000 UTC   contents:
2025-07-07 07:30:27.720254172 +0000 UTC     build repositories: [https://apk.cgr.dev/wolfi-presubmit/world-1751869916414 https://packages.wolfi.dev/os]
2025-07-07 07:30:27.720278989 +0000 UTC     runtime repositories: []
2025-07-07 07:30:27.72028783 +0000 UTC     keyring:      [https://packages.wolfi.dev/os/wolfi-signing.rsa.pub]
2025-07-07 07:30:27.720297165 +0000 UTC     packages:     [binutils=2.44-r9 build-base=1-r8 busybox=1.37.0-r46 ca-certificates-bundle=20241121-r42 cmake=4.0.3-r0 cyrus-sasl=2.1.28-r42 gcc=15.1.0-r1 gdbm=1.25-r2 git=2.50.0-r3 glibc-dev=2.41-r53 glibc-locale-posix=2.41-r53 glibc=2.41-r53 gmp=6.3.0-r7 heimdal-libs=7.8.0-r41 isl=0.27-r3 jsoncpp=1.9.6-r2 keyutils-libs=1.6.3-r35 krb5-conf=1.0-r6 krb5-libs=1.21.3-r42 ld-linux=2.41-r53 libacl1=2.3.2-r50 libarchive=3.8.1-r0 libatomic=15.1.0-r1 libbrotlicommon1=1.1.0-r5 libbrotlidec1=1.1.0-r5 libbz2-1=1.0.8-r18 libcom_err=1.47.2-r22 libcrypt1=2.41-r53 libcrypto3=3.5.1-r0 libcurl-openssl4=8.14.1-r2 libexpat1=2.7.1-r1 libgcc=15.1.0-r1 libgo=15.1.0-r1 libgomp=15.1.0-r1 libidn2=2.3.8-r2 libldap=2.6.10-r3 liblz4-1=1.10.0-r4 libnghttp2-14=1.66.0-r0 libpcre2-8-0=10.45-r2 libpsl=0.21.5-r5 libquadmath=15.1.0-r1 librhash0=1.4.6-r1 libssl3=3.5.1-r0 libstdc++-dev=15.1.0-r1 libstdc++=15.1.0-r1 libunistring=1.3-r3 libuv=1.51.0-r1 libverto=0.3.2-r5 libxcrypt-dev=4.4.38-r2 libxcrypt=4.4.38-r2 libzstd1=1.5.7-r2 linux-headers=6.15.4-r0 make=4.4.1-r7 mpc=1.3.1-r6 mpfr=4.2.2-r1 ncurses-terminfo-base=6.5_p20250621-r0 ncurses=6.5_p20250621-r0 nss-db=2.41-r53 nss-hesiod=2.41-r53 openssf-compiler-options=20240627-r23 patch=2.8-r2 pkgconf=2.5.1-r0 posix-cc-wrappers=1-r6 readline=8.3-r0 samurai=1.2-r3 scanelf=1.3.8-r4 sqlite-libs=3.50.2-r0 wolfi-baselayout=20230201-r21 xz=5.8.1-r1 zlib=1.3.1-r50]
2025-07-07 07:30:27.720336777 +0000 UTC   accounts:
2025-07-07 07:30:27.720356601 +0000 UTC     runas:  
2025-07-07 07:30:27.720367566 +0000 UTC     users:
2025-07-07 07:30:27.720378516 +0000 UTC       - uid=1000(build) gid=1000
2025-07-07 07:30:27.720398444 +0000 UTC     groups:
2025-07-07 07:30:27.720406553 +0000 UTC       - gid=1000(build) members=[build]
2025-07-07 07:30:27.720422861 +0000 UTC auth configured for: []
2025-07-07 07:30:27.925068424 +0000 UTC installing wolfi-baselayout (20230201-r21)
2025-07-07 07:30:27.989886585 +0000 UTC installing ca-certificates-bundle (20241121-r42)
2025-07-07 07:30:28.009421131 +0000 UTC installing glibc-locale-posix (2.41-r53)
2025-07-07 07:30:28.010298491 +0000 UTC installing libgcc (15.1.0-r1)
2025-07-07 07:30:28.347029407 +0000 UTC installing glibc (2.41-r53)
2025-07-07 07:30:28.34771181 +0000 UTC installing ld-linux (2.41-r53)
2025-07-07 07:30:28.34826363 +0000 UTC installing libstdc++ (15.1.0-r1)
2025-07-07 07:30:28.3486334 +0000 UTC installing libzstd1 (1.5.7-r2)
2025-07-07 07:30:28.810241399 +0000 UTC installing binutils (2.44-r9)
2025-07-07 07:30:28.810983588 +0000 UTC installing libxcrypt (4.4.38-r2)
2025-07-07 07:30:28.811348848 +0000 UTC installing libxcrypt-dev (4.4.38-r2)
2025-07-07 07:30:28.811779939 +0000 UTC installing linux-headers (6.15.4-r0)
2025-07-07 07:30:28.813516649 +0000 UTC installing nss-db (2.41-r53)
2025-07-07 07:30:28.814086945 +0000 UTC installing nss-hesiod (2.41-r53)
2025-07-07 07:30:29.058995793 +0000 UTC installing glibc-dev (2.41-r53)
2025-07-07 07:30:29.061021319 +0000 UTC installing libquadmath (15.1.0-r1)
2025-07-07 07:30:29.06156141 +0000 UTC installing libstdc++-dev (15.1.0-r1)
2025-07-07 07:30:29.064879746 +0000 UTC installing openssf-compiler-options (20240627-r23)
2025-07-07 07:30:29.065585492 +0000 UTC installing posix-cc-wrappers (1-r6)
2025-07-07 07:30:29.06585354 +0000 UTC installing libatomic (15.1.0-r1)
2025-07-07 07:30:29.066335546 +0000 UTC installing gmp (6.3.0-r7)
2025-07-07 07:30:29.066718829 +0000 UTC installing libgo (15.1.0-r1)
2025-07-07 07:30:29.067229127 +0000 UTC installing libgomp (15.1.0-r1)
2025-07-07 07:30:29.067872338 +0000 UTC installing isl (0.27-r3)
2025-07-07 07:30:29.068362505 +0000 UTC installing mpfr (4.2.2-r1)
2025-07-07 07:30:29.068883507 +0000 UTC installing mpc (1.3.1-r6)
2025-07-07 07:30:29.069361482 +0000 UTC installing zlib (1.3.1-r50)
2025-07-07 07:30:32.832039527 +0000 UTC installing gcc (15.1.0-r1)
2025-07-07 07:30:32.833988596 +0000 UTC installing make (4.4.1-r7)
2025-07-07 07:30:32.834386617 +0000 UTC installing pkgconf (2.5.1-r0)
2025-07-07 07:30:32.834778467 +0000 UTC installing build-base (1-r8)
2025-07-07 07:30:32.834918435 +0000 UTC installing libcrypt1 (2.41-r53)
2025-07-07 07:30:32.83540095 +0000 UTC installing busybox (1.37.0-r46)
2025-07-07 07:30:32.835782209 +0000 UTC installing libacl1 (2.3.2-r50)
2025-07-07 07:30:32.836114085 +0000 UTC installing libbz2-1 (1.0.8-r18)
2025-07-07 07:30:32.836433383 +0000 UTC installing libcrypto3 (3.5.1-r0)
2025-07-07 07:30:32.836852813 +0000 UTC installing libexpat1 (2.7.1-r1)
2025-07-07 07:30:32.837115552 +0000 UTC installing liblz4-1 (1.10.0-r4)
2025-07-07 07:30:32.837432442 +0000 UTC installing xz (5.8.1-r1)
2025-07-07 07:30:32.837984392 +0000 UTC installing libarchive (3.8.1-r0)
2025-07-07 07:30:32.838257732 +0000 UTC installing libbrotlicommon1 (1.1.0-r5)
2025-07-07 07:30:32.838579102 +0000 UTC installing libbrotlidec1 (1.1.0-r5)
2025-07-07 07:30:32.838876574 +0000 UTC installing krb5-conf (1.0-r6)
2025-07-07 07:30:32.839002435 +0000 UTC installing libcom_err (1.47.2-r22)
2025-07-07 07:30:32.839359542 +0000 UTC installing keyutils-libs (1.6.3-r35)
2025-07-07 07:30:32.839690479 +0000 UTC installing libssl3 (3.5.1-r0)
2025-07-07 07:30:32.840117255 +0000 UTC installing libverto (0.3.2-r5)
2025-07-07 07:30:32.840458895 +0000 UTC installing krb5-libs (1.21.3-r42)
2025-07-07 07:30:32.841033636 +0000 UTC installing gdbm (1.25-r2)
2025-07-07 07:30:32.841373999 +0000 UTC installing ncurses-terminfo-base (6.5_p20250621-r0)
2025-07-07 07:30:32.842294942 +0000 UTC installing ncurses (6.5_p20250621-r0)
2025-07-07 07:30:32.842652415 +0000 UTC installing readline (8.3-r0)
2025-07-07 07:30:32.842908518 +0000 UTC installing sqlite-libs (3.50.2-r0)
2025-07-07 07:30:32.843191447 +0000 UTC installing heimdal-libs (7.8.0-r41)
2025-07-07 07:30:32.843603333 +0000 UTC installing cyrus-sasl (2.1.28-r42)
2025-07-07 07:30:32.844034176 +0000 UTC installing libldap (2.6.10-r3)
2025-07-07 07:30:32.844472536 +0000 UTC installing libnghttp2-14 (1.66.0-r0)
2025-07-07 07:30:32.84481972 +0000 UTC installing libunistring (1.3-r3)
2025-07-07 07:30:32.845078497 +0000 UTC installing libidn2 (2.3.8-r2)
2025-07-07 07:30:32.845375617 +0000 UTC installing libpsl (0.21.5-r5)
2025-07-07 07:30:32.845724433 +0000 UTC installing libcurl-openssl4 (8.14.1-r2)
2025-07-07 07:30:32.846085982 +0000 UTC installing jsoncpp (1.9.6-r2)
2025-07-07 07:30:32.846199478 +0000 UTC installing librhash0 (1.4.6-r1)
2025-07-07 07:30:32.846509154 +0000 UTC installing libuv (1.51.0-r1)
2025-07-07 07:30:32.846845298 +0000 UTC installing cmake (4.0.3-r0)
2025-07-07 07:30:32.854711726 +0000 UTC installing libpcre2-8-0 (10.45-r2)
2025-07-07 07:30:32.855159297 +0000 UTC installing git (2.50.0-r3)
2025-07-07 07:30:32.856047586 +0000 UTC installing patch (2.8-r2)
2025-07-07 07:30:32.85627772 +0000 UTC installing samurai (1.2-r3)
2025-07-07 07:30:32.856398735 +0000 UTC installing scanelf (1.3.8-r4)
2025-07-07 07:30:40.230738539 +0000 UTC qemu: generating ssh key pairs for ephemeral VM
2025-07-07 07:30:40.232819251 +0000 UTC qemu: generating initramfs
2025-07-07 07:30:40.50956308 +0000 UTC image configuration:
2025-07-07 07:30:40.509620716 +0000 UTC   contents:
2025-07-07 07:30:40.509628684 +0000 UTC     build repositories: [https://apk.cgr.dev/chainguard]
2025-07-07 07:30:40.509639958 +0000 UTC     runtime repositories: []
2025-07-07 07:30:40.509645664 +0000 UTC     keyring:      []
2025-07-07 07:30:40.509650593 +0000 UTC     packages:     [microvm-init]
2025-07-07 07:30:45.61380101 +0000 UTC installing wolfi-baselayout (20230201-r21)
2025-07-07 07:30:45.615281948 +0000 UTC installing ca-certificates-bundle (20241121-r42)
2025-07-07 07:30:45.61613537 +0000 UTC installing libgcc (15.1.0-r1)
2025-07-07 07:30:45.616802609 +0000 UTC installing glibc-locale-posix (2.41-r53)
2025-07-07 07:30:45.618356045 +0000 UTC installing glibc (2.41-r53)
2025-07-07 07:30:45.644482832 +0000 UTC installing ld-linux (2.41-r53)
2025-07-07 07:30:45.645523523 +0000 UTC installing gnutar (1.35-r4)
2025-07-07 07:30:45.651324979 +0000 UTC installing libattr1 (2.5.2-r50)
2025-07-07 07:30:45.651936163 +0000 UTC installing attr (2.5.2-r50)
2025-07-07 07:30:45.652806191 +0000 UTC installing zlib (1.3.1-r50)
2025-07-07 07:30:45.653467245 +0000 UTC installing libzstd1 (1.5.7-r2)
2025-07-07 07:30:45.654834076 +0000 UTC installing xz (5.8.1-r1)
2025-07-07 07:30:45.657866021 +0000 UTC installing libcrypto3 (3.5.1-r0)
2025-07-07 07:30:45.740866895 +0000 UTC installing kmod (34.2-r40)
2025-07-07 07:30:45.742072756 +0000 UTC installing libbz2-1 (1.0.8-r18)
2025-07-07 07:30:45.803135061 +0000 UTC installing libelf (0.193-r1)
2025-07-07 07:30:45.804762433 +0000 UTC installing libmnl (1.0.5-r6)
2025-07-07 07:30:45.80545207 +0000 UTC installing libnftnl (1.2.9-r0)
2025-07-07 07:30:45.80692463 +0000 UTC installing iptables (1.8.11-r23)
2025-07-07 07:30:45.814678678 +0000 UTC installing iproute2 (6.15.0-r0)
2025-07-07 07:30:45.818663578 +0000 UTC installing libpcre2-8-0 (10.45-r2)
2025-07-07 07:30:45.8199047 +0000 UTC installing libsepol (3.8.1-r50)
2025-07-07 07:30:45.82190429 +0000 UTC installing libselinux (3.8.1-r50)
2025-07-07 07:30:45.834364364 +0000 UTC installing libxcrypt (4.4.38-r2)
2025-07-07 07:30:45.835241754 +0000 UTC installing libcrypt1 (2.41-r53)
2025-07-07 07:30:45.836009598 +0000 UTC installing linux-pam (1.7.1-r0)
2025-07-07 07:30:45.84845778 +0000 UTC installing openssh-keygen (10.0_p1-r1)
2025-07-07 07:30:45.849632987 +0000 UTC installing openssh-server-config (10.0_p1-r1)
2025-07-07 07:30:45.850280915 +0000 UTC installing openssh-server (10.0_p1-r1)
2025-07-07 07:30:45.853112356 +0000 UTC installing libblkid (2.41.1-r1)
2025-07-07 07:30:45.855148659 +0000 UTC installing libmount (2.41.1-r1)
2025-07-07 07:30:45.857276644 +0000 UTC installing mount (2.41.1-r1)
2025-07-07 07:30:45.858254116 +0000 UTC installing ncurses-terminfo-base (6.5_p20250621-r0)
2025-07-07 07:30:45.870632303 +0000 UTC installing ncurses (6.5_p20250621-r0)
2025-07-07 07:30:45.873077295 +0000 UTC installing setarch (2.41.1-r1)
2025-07-07 07:30:45.874105213 +0000 UTC installing libuuid (2.41.1-r1)
2025-07-07 07:30:45.874902243 +0000 UTC installing libfdisk (2.41.1-r1)
2025-07-07 07:30:45.877089985 +0000 UTC installing sqlite-libs (3.50.2-r0)
2025-07-07 07:30:45.878956602 +0000 UTC installing util-linux (2.41.1-r1)
2025-07-07 07:30:45.884229613 +0000 UTC installing libsmartcols (2.41.1-r1)
2025-07-07 07:30:45.886020897 +0000 UTC installing util-linux-misc (2.41.1-r1)
2025-07-07 07:30:45.90326733 +0000 UTC installing libstdc++ (15.1.0-r1)
2025-07-07 07:30:45.906414408 +0000 UTC installing inih (60-r0)
2025-07-07 07:30:45.907284165 +0000 UTC installing userspace-rcu (0.15.3-r0)
2025-07-07 07:30:45.910795837 +0000 UTC installing xfsprogs-core (6.15.0-r0)
2025-07-07 07:30:45.913322878 +0000 UTC installing xfsprogs (6.15.0-r0)
2025-07-07 07:30:45.914005684 +0000 UTC installing busybox (1.37.0-r46)
2025-07-07 07:30:45.91537361 +0000 UTC installing microvm-init (0.0.1-r11)
2025-07-07 07:30:46.986808799 +0000 UTC qemu: starting VM
2025-07-07 07:30:46.988129079 +0000 UTC qemu: waiting for SSH
2025-07-07 07:30:50.832306373 +0000 UTC qemu: meta-data=/dev/vda               isize=512    agcount=8, agsize=1638400 blks
2025-07-07 07:30:50.832423757 +0000 UTC qemu:          =                       sectsz=4096  attr=2, projid32bit=1
2025-07-07 07:30:50.832433744 +0000 UTC qemu:          =                       crc=1        finobt=1, sparse=1, rmapbt=1
2025-07-07 07:30:50.83244132 +0000 UTC qemu:          =                       reflink=1    bigtime=1 inobtcount=1 nrext64=1
2025-07-07 07:30:50.832447499 +0000 UTC qemu:          =                       exchange=0   metadir=0
2025-07-07 07:30:50.832453194 +0000 UTC qemu: data     =                       bsize=4096   blocks=13107200, imaxpct=25
2025-07-07 07:30:50.832461497 +0000 UTC qemu:          =                       sunit=0      swidth=0 blks
2025-07-07 07:30:50.832468469 +0000 UTC qemu: naming   =version 2              bsize=4096   ascii-ci=0, ftype=1, parent=0
2025-07-07 07:30:50.832476967 +0000 UTC qemu: log      =internal log           bsize=4096   blocks=16384, version=2
2025-07-07 07:30:50.832651456 +0000 UTC qemu:          =                       sectsz=4096  sunit=1 blks, lazy-count=1
2025-07-07 07:30:50.832697702 +0000 UTC qemu: realtime =none                   extsz=4096   blocks=0, rtextents=0
2025-07-07 07:30:50.832705687 +0000 UTC qemu:          =                       rgcount=0    rgsize=0 extents
2025-07-07 07:30:50.8327168 +0000 UTC qemu:          =                       zoned=0      start=0 reserved=0
2025-07-07 07:30:50.835444173 +0000 UTC qemu: Discarding blocks...Done.
2025-07-07 07:30:52.181521379 +0000 UTC qemu: ssh-keygen: generating new host keys: RSA ECDSA ED25519 
2025-07-07 07:30:52.191749392 +0000 UTC qemu: Server listening on 0.0.0.0 port 2223.
2025-07-07 07:30:52.191828893 +0000 UTC qemu: Server listening on 0.0.0.0 port 22.
2025-07-07 07:30:52.525106967 +0000 UTC qemu: VM started successfully, SSH server is up
2025-07-07 07:30:52.525403782 +0000 UTC qemu: Connection closed by 10.0.2.2 port 38958
2025-07-07 07:30:52.557273402 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 38968 ssh2: ECDSA SHA256:8LypsB3szI8oHtsh2Wms2cqoF6Qdm3sK2VWbcPA7TRA
2025-07-07 07:30:52.565485259 +0000 UTC qemu: Connection closed by 10.0.2.2 port 38968
2025-07-07 07:30:52.592838792 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 38974 ssh2: ECDSA SHA256:8LypsB3szI8oHtsh2Wms2cqoF6Qdm3sK2VWbcPA7TRA
2025-07-07 07:30:52.626317447 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 53220 ssh2: ECDSA SHA256:8LypsB3szI8oHtsh2Wms2cqoF6Qdm3sK2VWbcPA7TRA
2025-07-07 07:30:52.632679488 +0000 UTC qemu: setting up melange cachedir: ./melange-cache/
2025-07-07 07:30:52.650363572 +0000 UTC qemu: setting up local workspace
2025-07-07 07:30:52.680500555 +0000 UTC running step "git-checkout"
2025-07-07 07:30:52.696794004 +0000 UTC [git checkout] repo='https://github.com/google/crc32c' dest='.' depth='unset' branch='' tag='1.1.2' expcommit='02e65f4fd3065d27b2e29324800ca6d04df16126' recurse='false'
2025-07-07 07:30:52.703401085 +0000 UTC [git checkout] execute: git config --global --add safe.directory /tmp/tmp.ZDCaHG
2025-07-07 07:30:52.705547488 +0000 UTC [git checkout] execute: git config --global --add safe.directory /home/build
2025-07-07 07:30:52.707620969 +0000 UTC [git checkout] execute: git clone --quiet --origin=origin --config=user.name=Melange Build --config=user.email=melange-build@cgr.dev --config=advice.detachedHead=false --branch=1.1.2 --depth=1 https://github.com/google/crc32c /tmp/tmp.ZDCaHG
2025-07-07 07:30:53.014019564 +0000 UTC [git checkout] execute: cd /tmp/tmp.ZDCaHG
2025-07-07 07:30:53.014078267 +0000 UTC [git checkout] tar -c . | tar -C "/home/build" -x
2025-07-07 07:30:53.026443051 +0000 UTC [git checkout] execute: cd /home/build
2025-07-07 07:30:53.026496241 +0000 UTC [git checkout] execute: git config --global --add safe.directory /home/build
2025-07-07 07:30:53.028507282 +0000 UTC [git checkout] execute: git fetch --quiet origin --depth=1 --no-tags +refs/tags/1.1.2:refs/origin/tags/1.1.2
2025-07-07 07:30:53.255859845 +0000 UTC [git checkout] execute: git checkout --quiet origin/tags/1.1.2
2025-07-07 07:30:53.262956411 +0000 UTC [git checkout] tag 1.1.2 is 02e65f4fd3065d27b2e29324800ca6d04df16126
2025-07-07 07:30:53.264953575 +0000 UTC running step "patch"
2025-07-07 07:30:53.287182545 +0000 UTC Hmm...  Looks like a unified diff to me...
2025-07-07 07:30:53.287248908 +0000 UTC The text leading up to this was:
2025-07-07 07:30:53.287259814 +0000 UTC --------------------------
2025-07-07 07:30:53.287267091 +0000 UTC |Author: Holger Jaekel <holger.jaekel@gmx.de>
2025-07-07 07:30:53.2872746 +0000 UTC |Summary: ignore googletest, glog, and benchmark from the third-party folder
2025-07-07 07:30:53.287281001 +0000 UTC |----
2025-07-07 07:30:53.2872875 +0000 UTC |
2025-07-07 07:30:53.287293612 +0000 UTC |--- a/CMakeLists.txt
2025-07-07 07:30:53.28729942 +0000 UTC |+++ b/CMakeLists.txt
2025-07-07 07:30:53.287305484 +0000 UTC --------------------------
2025-07-07 07:30:53.287378086 +0000 UTC patching file CMakeLists.txt
2025-07-07 07:30:53.287422525 +0000 UTC Hunk #1 succeeded at 16.
2025-07-07 07:30:53.287431346 +0000 UTC Hunk #2 succeeded at 187.
2025-07-07 07:30:53.287437293 +0000 UTC Hunk #3 succeeded at 296.
2025-07-07 07:30:53.287443144 +0000 UTC Hunk #4 succeeded at 356.
2025-07-07 07:30:53.287448946 +0000 UTC done
2025-07-07 07:30:53.288132693 +0000 UTC running step "cmake/configure"
2025-07-07 07:30:53.411996877 +0000 UTC CMake Error at CMakeLists.txt:5 (cmake_minimum_required):
2025-07-07 07:30:53.412058664 +0000 UTC   Compatibility with CMake < 3.5 has been removed from CMake.
2025-07-07 07:30:53.412066861 +0000 UTC 
2025-07-07 07:30:53.412073304 +0000 UTC   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
2025-07-07 07:30:53.412077201 +0000 UTC   to tell CMake that the project requires at least <min> but has been updated
2025-07-07 07:30:53.412084304 +0000 UTC   to work with policies introduced by <max> or earlier.
2025-07-07 07:30:53.412087555 +0000 UTC 
2025-07-07 07:30:53.412091338 +0000 UTC   Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
2025-07-07 07:30:53.412094677 +0000 UTC 
2025-07-07 07:30:53.412097741 +0000 UTC 
2025-07-07 07:30:53.412236242 +0000 UTC -- Configuring incomplete, errors occurred!
2025-07-07 07:30:53.414487382 +0000 UTC Failed to run command "/bin/sh -c 'set -e 
[ -d '\''/home/build'\'' ] || mkdir -p '\''/home/build'\''
cd '\''/home/build'\''
cmake -B output -G Ninja \
	-DCMAKE_INSTALL_PREFIX=/usr \
	-DCMAKE_INSTALL_LIBDIR=lib \
	-DCMAKE_BUILD_TYPE=Release \
	-DCMAKE_VERBOSE_MAKEFILE=ON \
	-DBUILD_SHARED_LIBS=True \
	-DCRC32C_BUILD_TESTS="OFF" \
	-DCRC32C_BUILD_BENCHMARKS="OFF" \
	-DCRC32C_USE_GLOG="OFF" \
	-DCRC32C_INSTALL="ON"

exit 0'": Process exited with status 1
2025-07-07 07:30:53.414632024 +0000 UTC qemu: sending shutdown signal
2025-07-07 07:30:53.424146698 +0000 UTC failed to build package: unable to run package crc32c pipeline: unable to run pipeline: unable to run pipeline: Process exited with status 1
2025-07-07 07:30:53.511129246 +0000 UTC failed to build: command melange failed: exit status 1
2025-07-07 07:30:53.511402914 +0000 UTC build failed: command melange failed: exit status 1
```

--- a/crc32c.yaml
+++ b/crc32c.yaml
@@ -1,7 +1,7 @@
 package:
   name: crc32c
   version: 1.1.2
-  epoch: 2
+  epoch: 3
   description: CRC32C implementation with support for CPU-specific acceleration instructions
   copyright:
     - license: BSD-3-Clause
@@ -12,6 +12,7 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
+      - cmake-3
 
 pipeline:
   - uses: git-checkout

--- a/libcbor.yaml
+++ b/libcbor.yaml
@@ -1,7 +1,7 @@
 package:
   name: libcbor
   version: "0.12.0"
-  epoch: 1
+  epoch: 2
   description: CBOR protocol implementation for C
   copyright:
     - license: MIT
@@ -11,6 +11,7 @@ environment:
     packages:
       - build-base
       - busybox
+      - cmake-3
 
 pipeline:
   - uses: git-checkout

--- a/libsrt.yaml
+++ b/libsrt.yaml
@@ -1,7 +1,7 @@
 package:
   name: libsrt
   version: 1.5.4
-  epoch: 2
+  epoch: 3
   description: "Secure Reliable Transport (SRT)"
   copyright:
     - license: MPL-2.0
@@ -12,6 +12,7 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
+      - cmake-3
       - openssl
       - openssl-dev
       - wolfi-base

--- a/nfs-ganesha.yaml
+++ b/nfs-ganesha.yaml
@@ -1,7 +1,7 @@
 package:
   name: nfs-ganesha
   version: "6.5"
-  epoch: 3
+  epoch: 4
   description: NFS-Ganesha is an NFSv3,v4,v4.1 fileserver that runs in user mode on most UNIX/Linux systems
   copyright:
     - license: GPL-3.0-or-later
@@ -18,6 +18,7 @@ environment:
       - busybox
       - ca-certificates-bundle
       - ceph-dev
+      - cmake-3
       - flex
       - gcc-12-default
       - krb5-dev

--- a/orthanc-dicomweb.yaml
+++ b/orthanc-dicomweb.yaml
@@ -1,7 +1,7 @@
 package:
   name: orthanc-dicomweb
   version: 1.20
-  epoch: 0
+  epoch: 1
   description: DICOMweb plugin for Orthanc DICOM server - RESTful DICOM services
   copyright:
     - license: AGPL-3.0-or-later
@@ -18,6 +18,7 @@ environment:
       - boost-dev
       - build-base
       - busybox
+      - cmake-3
       - e2fsprogs
       - gtest-dev
       - jsoncpp-dev

--- a/orthanc-gdcm.yaml
+++ b/orthanc-gdcm.yaml
@@ -1,7 +1,7 @@
 package:
   name: orthanc-gdcm
   version: 1.8
-  epoch: 0
+  epoch: 1
   description: GDCM plugin for Orthanc DICOM server - advanced DICOM image decoding
   copyright:
     - license: AGPL-3.0-or-later
@@ -15,6 +15,7 @@ environment:
       - boost-dev
       - build-base
       - busybox
+      - cmake-3
       - e2fsprogs-dev
       - gdcm
       - gtest-dev

--- a/orthanc-ohif.yaml
+++ b/orthanc-ohif.yaml
@@ -1,7 +1,7 @@
 package:
   name: orthanc-ohif
   version: 1.6
-  epoch: 0
+  epoch: 1
   description: OHIF plugin for Orthanc DICOM server - modern DICOM web viewer
   copyright:
     - license: AGPL-3.0-or-later
@@ -20,6 +20,7 @@ environment:
       - boost-dev
       - build-base
       - busybox
+      - cmake-3
       - e2fsprogs-dev
       - gtest-dev
       - jsoncpp-dev

--- a/orthanc-webviewer.yaml
+++ b/orthanc-webviewer.yaml
@@ -1,7 +1,7 @@
 package:
   name: orthanc-webviewer
   version: "2.10"
-  epoch: 0
+  epoch: 1
   description: Web viewer plugin for Orthanc DICOM server - medical image visualization
   copyright:
     - license: AGPL-3.0-or-later
@@ -18,6 +18,7 @@ environment:
       - boost-dev
       - build-base
       - busybox
+      - cmake-3
       - e2fsprogs-dev
       - gtest-dev
       - jsoncpp-dev

--- a/plog.yaml
+++ b/plog.yaml
@@ -1,7 +1,7 @@
 package:
   name: plog
   version: 1.1.10
-  epoch: 1
+  epoch: 2
   description: "Portable, simple and extensible C++ logging library"
   copyright:
     - license: 'MIT'
@@ -11,6 +11,7 @@ environment:
     packages:
       - build-base
       - busybox
+      - cmake-3
 
 pipeline:
   - uses: git-checkout

--- a/soxr.yaml
+++ b/soxr.yaml
@@ -2,7 +2,7 @@
 package:
   name: soxr
   version: 0.1.3
-  epoch: 2
+  epoch: 3
   description: High quality, one-dimensional sample-rate conversion library
   copyright:
     - license: LGPL-2.1-or-later
@@ -15,6 +15,7 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
+      - cmake-3
 
 pipeline:
   - uses: fetch

--- a/yaml-cpp.yaml
+++ b/yaml-cpp.yaml
@@ -1,7 +1,7 @@
 package:
   name: yaml-cpp
   version: 0.8.0
-  epoch: 7
+  epoch: 8
   description: "yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec."
   copyright:
     - license: MIT
@@ -12,6 +12,7 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
+      - cmake-3
       - gcc-14-default
       - wolfi-base
 

--- a/zeromq.yaml
+++ b/zeromq.yaml
@@ -1,7 +1,7 @@
 package:
   name: zeromq
   version: 4.3.5
-  epoch: 4
+  epoch: 5
   description: The ZeroMQ messaging library and tools
   copyright:
     - license: MPL-2.0
@@ -10,6 +10,7 @@ environment:
   contents:
     packages:
       - build-base
+      - cmake-3
       - libbsd-dev
       - libsodium-dev
       - perl

